lib/main.dart

```diff
@@ -24,22 +24,25 @@ import 'services/services.dart';
 import 'utils/utils.dart';
 import 'widgets/widgets.dart';
 
-final AtSignLogger _logger = AtSignLogger(AtEnv.appNamespace);
+AtSignLogger _logger = AtSignLogger(AtEnv.appNamespace);
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
+  await AtEnv.load();
```

Comment:
This is already called on line 39, no need to call it again.

Reply:
Fixed